src/commands/mute/subcommands/add.ts

```diff
@@ -0,0 +1,45 @@
+import * as Discord from '../../../Discord';
+import SubCommand from '../../../SubCommand';
+import CommandOption from '../../../CommandOption';
+import { checkMutePermissions, mustGetRolesConfig } from '../_common';
+import { bold } from '../../../format';
+
+const add = new SubCommand({
+  name: 'add',
+  displayName: 'Add Mute',
+  description: 'Mutes a user',
+  requiredPermissions: [Discord.Permission.KICK_MEMBERS],
+  options: [
+    new CommandOption({
+      name: 'user',
+      description: 'The user to mute',
+      required: true,
+      type: Discord.CommandOptionType.USER,
+    }),
+  ],
+  handler: async ctx => {
+    const guildId = ctx.mustGetGuildId();
+    const targetUserId = ctx.getArgument<string>('user') as string;
+
+    if (!(await checkMutePermissions(ctx, guildId, targetUserId))) return;
```

Comment:
How about adding a response like "that user is already muted" instead of just returning?

Reply:
Currently `checkMutePermissions` contains a call to `respondWithError` when `false` is returned